### PR TITLE
chore(docs): update readme file

### PR DESCRIPTION
**Pull request by icfantv**
* add new sections using UIBS project as guide
* add stubs for sections with no content
* update doc from meeting minutes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,113 @@
...
-This project is being created by the [ui-bootstrap team](https://github.com/angular-ui/bootstrap).
+# Installation
+
+TBD
+
+#### Install with NPM
+
+TBD
+
+#### Install with Bower
```
Comment (Foxandxss) [inline README.md]: No bower.

File: README.md [hunk 2]
```diff
@@ -2,12 +2,113 @@
...
+
+TBD
+
+#### Install with NuGet
```
Comment (Foxandxss) [inline README.md]: Hopefully not this either.

File: README.md [hunk 3]
```diff
@@ -2,12 +2,113 @@
...
+
+TBD
+
+#### Custom build
```
Comment (Foxandxss) [inline README.md]: No custom builds.

File: README.md [hunk 4]
```diff
@@ -2,12 +2,113 @@
...
+
+TBD
+
+#### Manual download
+
+TBD
+
+### Adding dependency to your project
```
Comment (Foxandxss) [inline README.md]: Not this either.

File: README.md [hunk 5]
```diff
@@ -2,12 +2,113 @@
...
+
+As of this writing, Angular 2 browser support is unknown, but Bootstrap's are:
+
+* Chrome (stable and canary channel)
+* Firefox
+* IE 9+ (but see [Bootstrap 4's notes](http://v4-alpha.getbootstrap.com/getting-started/browsers-devices/#internet-explorer-9) on IE9)
+* Opera
+* Safari
+
+While modern mobile browsers should work without problems, we have made certain design decisions surrounding
```
Comment (Foxandxss) [inline README.md]: Please not this.

File: README.md [hunk 6]
```diff
@@ -2,12 +2,113 @@
...
+
+# Support
+
+## FAQ
+
+TBD
+
+## Code of Conduct
+
+Take a moment to read or [Code of Conduct](CODE_OF_CONDUCT.md)
```
Comment (Foxandxss) [inline README.md]: our

File: README.md [hunk 7]
```diff
@@ -2,12 +2,113 @@
...
+
+## Supported browsers
+
+As Bootstrap 4 and Angular 2 are still in active development this list of supported browsers is subject to
+change. It is our intention to support the same browsers and versions supported by both Bootstrap 4 and
+Angular 2, which ever is the _most_ restrictive.
+
+As of this writing, Angular 2 browser support is unknown, but Bootstrap's are:
```
Comment (Foxandxss) [inline README.md]: They are known.

**Comment by pkozlowski-opensource** [comment]
@icfantv I'm not big fan of leaving placeholders for things we _might_ add in the future. We can add content when we know what to add but I'm not sure we want to create placeholders for those.

**Comment by Foxandxss** [comment]
I totally agree with you Pawel. The original idea was to just add that we want to support X browsers and bootstrap 4. Nothing else, no placeholders.

We discussed whether or not to add stuff like "We are not going to support tooltips and popovers" and my response was no, because we have no idea how is this going to evolve, and you can be surprised.

I vote to just add the support browsers and bootstrap version bits and even so, it can change in the future I guess. For the rest of things, we will add more sections to the readme when they are needed.

**Comment by icfantv** [comment]
@pkozlowski-opensource, funny you say that.  I was telling @Foxandxss that we should add the tooltip/popover bit on modals to set expectations and it never occurred to me that adding placeholders could, even indirectly, set expectations.  I agree with you both there and will remove them.

I'd like another set of eyes on that bit for supporting tooltips/popovers on modals.  I think we either support it or we don't.  We've already made the decision in the UIBS project to **NOT** support them so I don't know that it makes sense to do so in this project.  But we should discuss.  @wesleycho, can you please also chime in?  Thanks.

**Comment by pkozlowski-opensource** [comment]
> I'd like another set of eyes on that bit for supporting tooltips/popovers on modals. I think we either support it or we don't. We've already made the decision in the UIBS project to NOT support them so I don't know that it makes sense to do so in this project.

Are those supported in the original bootstrap?

File: README.md (hunk 7 above)
Comment (wesleycho) [inline README.md]: They are not official yet - I think this might need some chiming in from @pkozlowski-opensource

**Comment by Foxandxss** [comment]
The problem Adam is that Angular 2 with webcomponents is a box of surprises. What in the past wasn't possible, now it is possible.

Not saying that tooltip/popover on mobiles (you mean mobiles and not modals, right?) is now a possibility, but we shouldn't close doors when we don't have a popover/tooltip yet.

File: README.md (hunk 5 above)
Comment (wesleycho) [inline README.md]: This whole passage is unnecessary - remove it.

File: README.md (hunk 7 above)
Comment (Foxandxss) [inline README.md]: Well, their README has some kind of browser support :P

**Comment by wesleycho** [comment]
I was not aware that we did not support tooltips and popovers in modals in UI Bootstrap. Was there a particular technical limitation there?

File: README.md (hunk 7 above)
Comment (pkozlowski-opensource) [inline README.md]: They _are_ known and _officially_ - things were announced during AngularConnect (keynote). You can see a list of browsers to be supported when you go to the main angular repo (SauceLabs badge)

**Comment by icfantv** [comment]
Gah!  Sorry.  I meant "mobile" not modal.  Will fix.

**Comment by icfantv** [comment]
@pkozlowski-opensource, see above comment.  Sorry for the confusion.  @Foxandxss, remember, ours was a UX decision, not an architecture one.  That is, we made it because we thought it was poor UX to have tooltips and popovers on mobile devices not because we couldn't do it from a code perspective.

File: README.md [hunk 8]
```diff
@@ -2,12 +2,113 @@
...
+
+#### Install with NuGet
+
+TBD
+
+#### Custom build
+
+TBD
+
+#### Manual download
```
Comment (wesleycho) [inline README.md]: Do we want this section too? The only reason I can think of to support this is for Plunkers and such, but we should strongly adopt npm as the main mechanism for consuming the library.

**Comment by pkozlowski-opensource** [comment]
@icfantv ok. So, I wouldn't worry about anything mobile at this point - it is way too early as we don't know how we are going to code tooltips / popovers nor what mobile support we are going to have from ng2.

**Comment by icfantv** [comment]
@pkozlowski-opensource.  ok.  I'll take it out.  I do feel like we should have a conversation eventually about cross-project consistency though.  Changes in architecture and what we can support is one thing, but design decisions made around UX (e.g., only supporting one dropdown open at a time), is another and shouldn't change just because the architecture does.  Right?

**Comment by wesleycho** [comment]
It's too early to mention anything specific IMO.

**Comment by Foxandxss** [comment]
As I said yesterday. We shouldn't get anything from ui-bootstrap for granted. We should question **EVERYTHING** we did in there and do it better now.

File: README.md (hunk 7 above)
Comment (icfantv) [inline README.md]: Sorry, I was going by our meeting notes from early October which is why I missed it.

**Comment by icfantv** [comment]
Ok fellas.  Rev 2 is posted for your perusal.  I should note that several hours of therapy were necessary after the last review.

File: README.md [hunk 9]
```diff
@@ -1,13 +1,51 @@
...
+* Firefox (40+)
+* IE (9+) (but see [Bootstrap 4's notes](http://v4-alpha.getbootstrap.com/getting-started/browsers-devices/#internet-explorer-9) on IE9)
+* Edge (20+)
+* Safari (7+)
+
+## Contributing to the project
+
+We are always looking for the quality contributions! Please check the [CONTRIBUTING.md](CONTRIBUTING.md) doc for contribution guidelines.
+
+## Development, meeting minutes, roadmap and more.
```
Comment (Foxandxss) [inline README.md]: I don't think this adds any value.

Comment (icfantv) [inline README.md]: Sorry, the comments in the diff aren't clear.  Which line/section are you referring to?

Comment (Foxandxss) [inline README.md]: The Developmeng, meeting minutes section.

Comment (icfantv) [inline README.md]: Ok.  Will remove.

File: README.md [hunk 10]
```diff
@@ -1,13 +1,46 @@
...
 
-As for demos, we have a list of Plunkers with our components:
+As with Bootstrap 4 and Angular 2, this library is a work in progress. Please check out our list of
+[issues](https://github.com/ng-bootstrap/core/issues) to see all the things we are implementing.
+Feel free to make comments there.
+
+This project is being created by the [ui-bootstrap team](https://github.com/angular-ui/bootstrap).
+
+# Quick links
+- [Demo](#demo)
```
Comment (Foxandxss) [inline README.md]: space in here.

Comment (icfantv) [inline README.md]: sorry, where?   what space?

Comment (Foxandxss) [inline README.md]: Between #quick links and the first entry of the TOC

File: README.md [hunk 11]
```diff
@@ -1,13 +1,46 @@
...
-This project is being created by the [ui-bootstrap team](https://github.com/angular-ui/bootstrap).
+# Support
+
+## Code of Conduct
+
+Please take a moment and read our [Code of Conduct](CODE_OF_CONDUCT.md)
+
+## Supported browsers
+
+We support the same browsers and versions supported by both Bootstrap 4 and Angular 2, which ever is _more_ restrictive.
```
Comment (Foxandxss) [inline README.md]: whichever

File: README.md [hunk 12]
```diff
@@ -1,13 +1,47 @@
-### NG Bootstrap - [Angular 2](http://angular.io/) directives specific to [Bootstrap](http://getbootstrap.com)
+# NG Bootstrap - [Angular 2](http://angular.io/) directives specific to [Bootstrap](http://getbootstrap.com)
```
Comment (sirajc) [inline README.md]: For Bootstrap4, we should link to http://v4-alpha.getbootstrap.com/

**Pull request closed without merging**